### ISSUE TITLE
Add a way to recompute the indicator fields that are set to be recomputed daily

### ISSUE THREAD

**Issue by jeremi** [openspp-project/openspp-registry#216]
- We could have a parameter on the field that specifies this, and a daily cron that run the update
- Add the recompute daily boolean field in the custom fields UI

For fields meant to be updated on a daily basis, it's proposed to introduce a specific parameter, effectively marking them for daily reevaluation. This would be seamlessly facilitated by a daily cron job, which will scan and promptly update these marked fields. To streamline the user experience and allow administrators to easily set which fields require daily updates, a new 'Recompute Daily' boolean option will be integrated into the custom fields UI. This approach ensures that relevant data remains fresh and up-to-date, without the need for manual intervention.

![image](https://github.com/OpenSPP/openspp-registry/assets/2864612/29686c1c-1514-4f39-8418-3c0a23d5cd85)

### How to use
1. Go to Settings
2. Database Structure by clicking (+) button on the menu
3. You can't create fields but select one of them to computing dialy. 
4. You can use filter to select Dialy Recompute.
5. Further, you can filter by "z_ind" for a model.

**Comment by dasunhegoda** [comment]
- Attribute to identify the fields
- Create a cron job to update the indicators

**Comment by jeremi** [comment]
Was this implemented?

**Comment by dasunhegoda** [comment]
Yes, it's implemented @jeremi. PR: https://github.com/OpenSPP/openspp-modules/pull/251
